migrate.sh

```diff
@@ -43,8 +43,24 @@ echo "Destination Directory: $destination_dir"
 echo "" # Blank line to separate from any failure output
 
 # Convert source and destination directories to absolute paths
-source_dir=$(readlink -f "$source_dir")
-destination_dir=$(readlink -f "$destination_dir")
+source_dir=$(readlink -f "$source_dir") || { echo "Error: Failed to resolve source directory path"; exit 1; }
```

Comment:
I had some trouble running the script, where these lines were failing silently. I think adding these print statements might make it easier for users to troubleshoot what is going wrong. In my case the dest directory didn't exist. Lmk if there's a better way of handling this